django/db/backends/base/creation.py

```diff
@@ -27,6 +32,21 @@ def _nodb_cursor(self):
     def log(self, msg):
         sys.stderr.write(msg + os.linesep)
 
+    def mark_tests(self):
+        tests_map = (
```

Comment:
It might work to reuse the code but currently there's the drawback that a "skip reason" isn't set.